core-feature-pack/src/main/resources/configuration/standalone/subsystems.xml

```diff
@@ -5,5 +5,6 @@
       <subsystem>logging.xml</subsystem>
       <subsystem>jmx.xml</subsystem>
       <subsystem>request-controller.xml</subsystem>
+      <subsystem>security-manager.xml</subsystem>
```

Comment:
ah yes, the -secmgr. So I guess this should be fine as for SM enablement goes.
but still I don't think we should have this in default standalone.xml of shipped core.
That part should stay in full shipped configs as it was till now.